python/ray/worker.py

```diff
@@ -2286,6 +2299,7 @@ def _try_to_compute_deterministic_class_id(cls, depth=5):
 
 
 def register_custom_serializer(cls,
+                               driver_id=None,
```

Comment:
Can you move this argument to after `local` and before `worker`? I would prefer to avoid this argument since it shouldn't be used by the user and `register_custom_serializer` is part of the API. However, it's a bit better if the argument is at the end (similar to `worker`, which is also not used by the user).

Reply:
make sense